src/Language/Fixpoint/Types/Names.hs

```diff
@@ -44,6 +44,7 @@ module Language.Fixpoint.Types.Names (
   , dropSym
   , headSym
   , lengthSym
+  , mappendSym
```

Comment:
@ranjitjhala I cannot because symbolSyffix inserts a separator. I created another function in Names that does what I want, and do nor expose mappendSym, if this makes any difference